tests/test_input.py

```diff
@@ -85,26 +85,27 @@ def test_locate_file_valid_paths1(self):
         """Test that `config.locate_file` works with absolute paths."""
         config_abs_path = config_dir / "test_config_parsing_relative_path1.cfg"
         open(config_abs_path, "w").close()
-        self.assertEqual(
-            locate_file(config_abs_path, tests_dir),
-            str(config_dir / "test_config_parsing_relative_path1.cfg"),
-        )
+        computed_path = locate_file(config_abs_path, tests_dir)
+        expected_path = str(config_dir / "test_config_parsing_relative_path1.cfg")
+        self.assertEqual(Path(computed_path).resolve(), Path(expected_path).resolve())
```

Comment:
```suggestion
        computed_path = Path(locate_file(config_abs_path, tests_dir))
        expected_path = config_dir / "test_config_parsing_relative_path1.cfg"
        self.assertEqual(computed_path.resolve(), expected_path.resolve())
```
Since these are new and they have "path" in the name, maybe just make them Paths, particularly since you're currently casting a Path to a string, then casting that string to a Path in the next line.

Reply:
Of course! Thanks for catching this!